src/components/JobsPanelParameters/JobsPanelParametersView.js

```diff
@@ -74,6 +79,8 @@ const JobsPanelParametersView = ({
                   label="Name"
                   className="input-row__item"
                   floatingLabel
+                  required={nameNotValid(parametersState.newParameter.name)}
+                  requiredText="Names already exists"
```

Comment:
Please change to singular:
```suggestion
                  requiredText="Name already exists"
```